### PR TITLE
feat: add PHP 8.1 and PHP 8.2 docker images

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: php/cloudbuild-test.yaml [hunk 1]
```diff
@@ -15,32 +15,33 @@
...
+  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php80', '.']
+  dir: 'php/php80'
   waitFor: ['-']
 
-# PHP 7.4
+# PHP 8.1
 - name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php74', '.']
-  dir: 'php/php74'
+  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php80', '.']
```
Comment (codyoss) [inline php/cloudbuild-test.yaml:32]: The comment above is incorrect or some of the config here. Same for 8.2

File: php/cloudbuild.yaml [hunk 2]
```diff
@@ -39,15 +27,28 @@ steps:
...
-# PHP 7.3 / 7.4 / 8.0
+# PHP 8.0
 - name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php', '.']
-  dir: 'php/php'
+  args: ['build', '-t', 'gcr.io/cloud-devrel-kokoro-resources/php81', '.']
+  dir: 'php/php81'
   waitFor: ['-']
 
+# PHP 8.0
```
Comment (codyoss) [inline php/cloudbuild.yaml:36]: nit: 8.2

**Review by codyoss — APPROVED** [review]
A couple nits, but LGTM

File: php/cloudbuild.yaml [hunk 3]
```diff
@@ -39,15 +27,28 @@ steps:
   dir: 'php/php80'
   waitFor: ['-']
 
-# PHP 7.3 / 7.4 / 8.0
+# PHP 8.0
```
[the remark this answers is not in the document]
Comment (bshaffer) [reply php/cloudbuild.yaml:30]:
```suggestion
# PHP 8.1
```

File: php/cloudbuild.yaml (hunk 2 above)
Comment (bshaffer) [reply php/cloudbuild.yaml:36]:
```suggestion
# PHP 8.2
```

**Comment by bshaffer** [comment]
This is failing due to `Step #1: make: *** [Makefile:1137: ext/fileinfo/libmagic/apprentice.lo] Error 1`, which a google search says is an OOM issue. I could not find where to configure the machine-type (although, the readme implies that this is using `--machine-type=n1-highcpu-8`, which should be plenty). 

Where is the machine-type configured?

**Comment by bshaffer** [comment]
> Where is the machine-type configured?

I added this using `options.machineType` in the `cloudbuild.yaml` files

**Pull request merged**